src/cryptography/x509/name.py

```diff
@@ -73,6 +81,28 @@ def _escape_dn_value(val: str) -> str:
     return val
 
 
+def _parse_dn_value(val: str) -> str:
+    if not val:
+        return ""
+
+    # See https://tools.ietf.org/html/rfc4514#section-2.4
+    val = val.replace("\\\\", "\\")
```

Comment:
Shouldn't this be last?

Reply:
No, however the link above should link to section 3, not 2.4, which I've updated. In there, it states:

> If in <string> form, a LDAP string representation asserted value can
> be obtained by replacing (left to right, non-recursively) each <pair>
> appearing in the <string> as follows:
> 
>     replace <ESC><ESC> with <ESC>;
>     replace <ESC><special> with <special>;
>     replace <ESC><hexpair> with the octet indicated by the <hexpair>.

Reply:
That's counter to the way that escaping is handled in most protocols.